quinn-h3/examples/h3.rs

```diff
@@ -268,20 +271,62 @@ fn client(
                 .map_err(|e| format_err!("send request: {}", e))
                 .and_then(|response| {
                     println!("received headers: {:?}", response.response());
-                    response.body().map_err(|e| format_err!("recv body: {}", e))
-                })
-                .and_then(|(data, trailers)| {
-                    println!("received body: {:?}", data);
-                    if let Some(trailers) = trailers {
-                        println!("received trailers: {:?}", trailers);
+                    Process {
+                        state: ProcessState::Body(response.body_stream()),
                     }
-                    futures::future::ok(())
+                    .map_err(|e| format_err!("send request: {}", e))
                 })
         });
 
     Ok((endpoint_driver, Box::new(fut)))
 }
 
+enum ProcessState {
+    Body(RecvBodyStream),
+    Trailers(DecodeHeaders),
+    Finished,
+}
+
+struct Process {
```

Comment:
Where does the term `Process` come from? Is it in the spec? If not, what kind of processing does it represent? As a name it feels very generic and therefore not as helpful as it could be.

Reply:
I got this renamed to `ProcessResponse`.